videos/WoT-Explainer-Video/WOTMotion.de.vtt

```diff
@@ -0,0 +1,251 @@
+die Interoperabilitätsprobleme,
+
+13
+00:00:45.500 --> 00:00:49.000
+die bei einem früheren IoT-Projekt 
+aufgetreten sind. 
+
+14
+00:00:49.500 --> 00:00:53.000
+Nach einer Online-Recherche 
+für eine gute Lösung,
+
+15
+00:00:53.500 --> 00:00:58.000
+finden die Entwickler Informationen 
+über das Web of Things, WoT.
+
+16
+00:00:58.500 --> 00:01:03.000
+WoT ist ein W3C-Standard, welcher es Web-Entwicklern ermöglicht 
+IoT-Entwickler zu werden,
+
+17
+00:01:03.500 --> 00:01:08.000
+indem sie die Werkzeuge und Frameworks 
+wiederverwenden, die sie lieben. 
+
+18
+00:01:08.500 --> 00:01:12.000
+WoT kann auf viele verschiedene 
+IoT-Anwendungsdomänen angewendet werden,
+
+19
+00:01:12.500 --> 00:01:15.000
+wie z.B. in der Industrie- 
+oder Gebäudeautomation,
+
+20
+00:01:15.500 --> 00:01:19.000
+aber auch für die Herausforderung unseres Landwirtes 
+im Bereich der Landwirtschaft.
+
+21
+00:01:19.500 --> 00:01:23.000
+Die Entwickler sind glücklich 
+viele Informationen zu finden
+
+22
+00:01:23.500 --> 00:01:27.000
+von WoT-Tutorials wie auf thingweb.io, 
+YouTube, und GitHub.
+
+23
+00:01:27.500 --> 00:01:29.000
+Mit node-wot können verschiedene 
+Protokollanbindungen
+
+24
+00:01:29.500 --> 00:01:34.000
+wie HTTP, MQTT, CoAP 
+und Modbus,
+
+25
+00:01:34.500 --> 00:01:39.000
+sowie einem Browser-Bundle, können die Entwickler leicht eine 
+eine Full-Stack-IoT-Lösung entwickeln.
+
+26
+00:01:39.500 --> 00:01:42.000
+Ihre Lösung verbindet 
+alle Geräte
+
+27
+00:01:42.500 --> 00:01:45.000
+und beinhaltet eine großartige Benutzeroberfläche 
+die mit Web-Technologie aufgebaut ist.
+
+28
+00:01:45.500 --> 00:01:49.000
+Anstatt eine einmalige 
+proprietäre Lösung zu schaffen,
+
+29
+00:01:49.500 --> 00:01:54.000
+verwenden die Entwickler die WoT-Methodik 
+um IoT-Builiding Blocks zu erstellen.
+
+30
+00:01:54.500 --> 00:01:59.000
+Diese Building Blocks erweitern 
+ihr bestehendes Portfolio an Web-Technologien.
+
+31
+00:01:59.500 --> 00:02:03.000
+Thing Description, 
+WoT-Anwendungsvorlagen,
+
+32
+00:02:03.500 --> 00:02:08.000
+und IoT-Semantik sind jetzt im gleichen Portfolio 
+von wiederverwendbaren Modulen
+
+33
+00:02:08.500 --> 00:02:13.000
+wie HTML-Seiten, JavaScript-Pakete 
+oder CSS aus früheren Web-Projekten.
+
+34
+00:02:13.500 --> 00:02:18.000
+Das Hauptelement 
+der WoT-Methodik
+
+35
+00:02:18.500 --> 00:02:21.000
+ist die Thing Description, 
+auch bekannt als TD.
+
+36
+00:02:21.500 --> 00:02:26.000
+Die TD ist vergleichbar 
+mit einer index.html für Things. 
+
+37
+00:02:21.500 --> 00:02:30.000
+Verschiedene Fragen können beantwortet werden
+ durch die Informationen in der TD,
+
+38
+00:02:30.500 --> 00:02:40.000
+wie Identifikation, Datenorganisation, Zugriff, Einheiten,
+Typen, Interaktionen, Sicherheit oder Beziehungen.
+
+39
+00:02:40.500 --> 00:02:44.000
+Klassische IoT-Projekte 
+verursachen in der Regel hohe Kosten,
+
+40
+00:02:44.500 --> 00:02:47.000
+insbesondere aufgrund 
+durch den hohen Integrationsaufwand
```

Comment:
des hohen Integrationsaufwandes  ?

Reply:
"insbesondere aufgrund (durch) des hohen Integrationsaufwandes"

I would rather say it without "durch" but I don't really mind...